src/test/kotlin/org/rust/ide/typing/RustRawLiteralHashesDeleterTest.kt

```diff
@@ -0,0 +1,27 @@
+package org.rust.ide.typing
+
+class RustRawLiteralHashesDeleterTest : RustTypingTestCaseBase() {
+    override val dataPath = "org/rust/ide/typing/rawHashes/fixtures"
+
+    // Swap file names, so we can reuse tests for inserter :-D
+    override fun beforeFileName(fileName: String): String {
+        return super.afterFileName(fileName)
+    }
+
+    override fun afterFileName(fileName: String): String {
+        return super.beforeFileName(fileName)
+    }
```

Comment:
Like this:

```
fun doTest1(before: @Languge("Rust), String, after: @Languge("Rust), String) = inner(before, after)
fun doTest2(before: @Languge("Rust), String, after: @Languge("Rust), String) = inner(after, before)
```

(Github, why have you make "Start a review" a default button :sob: )